src/components/Home/Home.js

```diff
@@ -103,7 +103,15 @@ class Home extends React.Component {
                         fontSize: lerp(showAppsProgress, 37, 22) + 'px',
                       }}
                     >
-                      Welcome to Aragon 0.5 − The Architect
+                      Welcome to Aragon!
+                    </Text>
+                    <br></br>
+                    <Text
+                      style={{
+                        fontSize: lerp(showAppsProgress, 18, 14) + 'px',
+                      }}
+                    >
+                      You are interacting with {daoName}
```

Comment:
One thing to note is that this "name" could be an address or an ENS name, and currently, we don't create the reverse-mapping for the name so we can't reverse-resolve names.

Reply:
Great, only showing it if it's an ENS address now